main.tf

```diff
@@ -208,7 +208,7 @@ resource "null_resource" "gcloud_auth_google_credentials" {
   }
 }
 
-resource "null_resource" "run_command" {
+resource "null_resource" "run_upsert_command" {
```

Comment:
We shouldn't rename the command (as it will cause an unnecessary state migration).

Reply:
Makes sense!